src/main/java/net/openhft/chronicle/queue/impl/table/SingleTableStore.java

```diff
@@ -295,11 +296,23 @@ public synchronized LongValue acquireValueFor(CharSequence key, final long defau
         }
     }
 
+    @Nullable
+    private static ValueIn readEventIfNameEquals(WireIn wireIn, CharSequence expected) {
+        try (ScopedResource<StringBuilder> stlSb = Wires.acquireStringBuilderScoped()) {
+            StringBuilder sb = stlSb.get();
+            final ValueIn valueIn = wireIn.readEventName(sb);
+            if (StringUtils.equalsCaseIgnore(expected, sb)) {
+                return valueIn;
+            }
+            return null;
+        }
+    }
```

Comment:
And again

Reply:
But of course they all have slight variations, e.g.
- return null or throw exception when name doesn't match
- case sensitivity of the comparison
- the error type/message thrown